src/views/catalog/CreateFromInstanceTypes/components/CreateVMFooter/components/ActionButtons/ActionButtons.tsx

```diff
@@ -49,7 +50,16 @@ const ActionButtons: FC<ActionButtonsProps> = ({
   }, [activeNamespace, navigate, vmName]);
 
   const { disableButtonTooltipContent, isCreationDisabled, isViewYAMLDisabled } =
-    useStatusActionButtons(isSubmitting);
+    useStatusActionButtons(isLoading);
+
+  const onView = () => {
+    setLoadingYAML(true);
+    try {
+      onViewYAML();
```

Comment:
`onViewYAML` is async but it's called from ordinary callback so will return immediately.
If we really need to wait the we need to convert the flow to async as well. We might also make `onViewYAML` an ordinary method (it only calls `createModal`)